tags/faq/newtext.ytag

```diff
@@ -0,0 +1,16 @@
+type: text
+
+---
+
+Minecraft 1.19 changed how text components work. They are now created with methods on the `Text` class instead of constructors of subclasses.
```

Comment:
```suggestion
Minecraft 1.19 changed how text components work. They are now created with methods on the `Text` interface instead of constructors of subclasses.
```